test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/Assert.cs

```diff
@@ -313,7 +313,7 @@ public static void NuspecDoesNotContain(MSBuildResult result, string nuspecPath,
             }
         }
 
-        public static void NupkgContains(MSBuildResult result, string nupkgPath, string filePath)
+        public static void NupkgContains(MSBuildResult result, string nupkgPath, params string[] filePaths)
```

Comment:
Can you just keep this as-is and call it multiple times? We don't really do this with the other asserts.

Reply:
We unzip it to a fixed path which was causing multiple invocations to fail. `ExtractToDirectory` doesn't have a overwrite flag. This seemed like the least obtrusive solution. I can come up with something different if you feel strongly about this

Reply:
OH ok fine 👍 Add a comment